lib/types/stats.dart

```diff
@@ -8,10 +8,10 @@ class Stats {
   Stats(this.stats24h, this.stats14d);
   factory Stats.fromJson(Map<String, dynamic> json) => _$StatsFromJson(json);
 
-  @JsonKey(name: '24h')
+  @JsonKey(name: '24h', fromJson: Stat.fromJsonList, toJson: Stat.toJsonList)
   final List<Stat>? stats24h;
```

Comment:
Shouldn't the `Stat`class have a `@JsonSerializable` attribute? 
If that's present, you won't need to add the methods manually like you did, and it should just work

Reply:
Ah I see, it corresponds to an array and not to an object, nevermind then.

Reply:
Yeah, it's because we have this array of arrays that we have custom serialization.